README.md

```diff
@@ -1,17 +1,25 @@
-# civil-disobedience-toolkit
+# Civil-Disobedience-Toolkit
 
 #Purpose
-The purpose of the civil disobedience toolkit is to gather a collection of open source tools that enable free speech and civil disobedience. Currently therer are several technological tools used to enable mass surveillance of peaceful protests as
-well as stifle dissent. Tools such as stingrays to capture cell phone data,  unmanned aerial vehicles (drones), surveillance
+The purpose of the Civil Disobedience Toolkit is to gather a collection of open source tools that enable free speech and civil disobedience. Currently there are several technological tools used to enable mass surveillance of peaceful protests as
+well as stifle dissent. Tools such as stingrays to capture cell phone data, unmanned aerial vehicles (drones), surveillance
 over social media by creating fake social media accounts to monitor political activities of activists on social media. 
 
 #Focus Areas
-1. Enabling secure communications - Allowing activists to use secure encrypted means of communication to prevent mass surveillanve
+1. Enabling secure communications - Allowing activists to use secure encrypted means of communication to prevent mass surveillance
+ * Vitual Private Networks (VPN)
```

Comment:
_Virtual_ instead of _Vitual_

You might want to provide a link about the importance of VPNs, some people might not know. The EFF has a good doc on it [[link]](https://ssd.eff.org/en/module/choosing-vpn-thats-right-you).